src/swagger.yaml

```diff
@@ -0,0 +1,125 @@
+openapi: 3.0.0
+info:
+  title: Simple Weather API
+  description: A simple weather API used to provide weather services to user
+  contact:
+    email: wangshiyao96@gmail.com
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+
+servers:
+- url: https://localhost:8000
```

Comment:
Similar to before, would be a good idea to keep this on port 3000 for consistency.